pycm/pycm_curve.py

```diff
@@ -63,12 +67,27 @@ def lambda_fun(x): return threshold_func(
                 for item in CURVE_PARAMS:
                     data_temp[item].append(getattr(cm, item)[c])
             self.data[c] = data_temp
-        self.plot_x_axis = None
-        self.plot_y_axis = None
+        self.auc = None
+        self.plot_x_axis = self.data[self.classes[0]]["FPR"]
+        self.plot_y_axis = self.data[self.classes[0]]["TPR"]
 
-    def area(self):
-        """Area method."""
-        pass
+    def area(self, method="trapezoidal"):
+        """
+        Compute Area Under Curve (AUC) using trapezoidal or midpoint numerical integral technique.
+
+        :param method: numerical integral technique (trapezoidal or midpoint)
+        :type method: str
+        :return: Area Under Curve (AUC) value as float
+        """
+        x = self.plot_x_axis
+        y = self.plot_y_axis
+        if method == "trapezoidal":
+            self.auc = __trapezoidal_numeric_integral__(x, y)
+        elif method == "midpoint":
+            self.auc = __midpoint_numeric_integral__(x, y)
+        else:
+            pycmCurveError(AREA_METHOD_ERROR)
+        return self.auc
```

Comment:
Here, `self.auc` should be a dictionary instead of a float number (AUC of all classes).
Example:

```python
{"0":0.767, "1":0.989, ...}
```

This will help us to support multi-class mode.

Another option is to add a parameter like `class_name` to the `area` method to help users to calculate AUC for each class separately. (I think the first method that I mentioned above is better)